scss/_body_common.scss

```diff
@@ -15,32 +15,39 @@
 
 /// Small text
 @mixin oTypographySmall {
```

Comment:
@expe I'm not really into `<small>` tags as imo they fall into the same category as `<b>` and `<i>` in that they violate the separation of style and content.